janusgraph-bigtable/pom.xml

```diff
@@ -0,0 +1,37 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.janusgraph</groupId>
+        <artifactId>janusgraph</artifactId>
+        <version>0.2.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <artifactId>janusgraph-bigtable</artifactId>
+    <name>JanusGraph-Bigtable: Google Cloud Bigtable driver</name>
+    <url>http://janusgraph.org</url>
+
+    <properties>
+        <top.level.basedir>${basedir}/..</top.level.basedir>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.google.cloud.bigtable</groupId>
+            <artifactId>bigtable-hbase-1.x-shaded</artifactId>
+            <version>${bigtable.version}</version>
+            <exclusions>
+                  <exclusion>
+                       <groupId>org.apache.hbase</groupId>
+                       <artifactId>hbase-shaded-client</artifactId>
+                  </exclusion>
+            </exclusions>
+        </dependency>
+
+        <dependency>
+           <groupId>io.netty</groupId>
+           <artifactId>netty-tcnative-boringssl-static</artifactId>
```

Comment:
Is this dependency needed here?  From the links here, it does not seem to be.
https://cloud.google.com/bigtable/docs/using-maven
https://cloud.google.com/bigtable/docs/upgrading-clients

Reply:
You are right, it's no longer necessary.

Reply:
Removed `netty-tcnative-boringssl-static`.